pando-rt/src/global_ptr.cpp

```diff
@@ -79,7 +79,7 @@ void* asNativePtr(GlobalAddress globalAddr) noexcept {
 #if defined(PANDO_RT_USE_BACKEND_PREP)
 
   // yield to other hart and then issue the operation
-  hartYield();
+  //hartYield();
```

Comment:
assuming these were needed due to the subpar queue

Reply:
That is my hypothesis and it seems to be borne out, there is another yield that exists after calling executeOn. Removing that seems to torpedo perf. I can't figure out why.

Reply:
strange stuff